### PR TITLE
IOS-92: Enlarge post action buttons at larger text sizes

### PR DESCRIPTION
**Pull request by j-f1**
The share icon does not appear aligned with the boost button because the images are slightly different widths. ¯\\\_(ツ)\_/¯

### CODE REVIEW

File: MastodonSDK/Package.resolved [hunk 1]
```diff
@@ -72,15 +72,6 @@
...
     },
-    {
-      "identity" : "kingfisher",
-      "kind" : "remoteSourceControl",
-      "location" : "https://github.com/onevcat/Kingfisher.git",
-      "state" : {
-        "revision" : "44e891bdb61426a95e31492a67c7c0dfad1f87c5",
-        "version" : "7.4.1"
-      }
-    },
```
Comment (zeitschlag) [inline MastodonSDK/Package.resolved]: These two changes (removing `kingfisher` and `thirdpartymailer` from `Package.resolved`) is just cleanup, right?

File: MastodonSDK/Sources/MastodonUI/View/Control/ActionToolbarContainer.swift [hunk 2]
```diff
@@ -158,22 +127,9 @@ extension ActionToolbarContainer {
         case reply
         case reblog
         case like
-        case bookmark
```
Comment (zeitschlag) [inline MastodonSDK/Sources/MastodonUI/View/Control/ActionToolbarContainer.swift:161]: What's the need to remove this?

File: MastodonSDK/Sources/MastodonUI/View/Control/ActionToolbarContainer.swift [hunk 3]
```diff
@@ -85,7 +78,8 @@ extension ActionToolbarContainer {
             button.setTitle("", for: .normal)
             button.setTitleColor(.secondaryLabel, for: .normal)
             button.expandEdgeInsets = UIEdgeInsets(top: -10, left: -10, bottom: -10, right: -10)
-            button.setInsets(forContentPadding: .zero, imageTitlePadding: style.buttonTitleImagePadding)
+            button.setInsets(forContentPadding: .zero, imageTitlePadding: 4)
+            button.showsLargeContentViewer = true
```
Comment (zeitschlag) [inline MastodonSDK/Sources/MastodonUI/View/Control/ActionToolbarContainer.swift]: The `showsLargeContentViewer` seems to be a workaround, right?

What I would have expected from this issue/PR is Dynamic Type-support for the four buttons (reply, reblog, favorite, share). Not that `UILargeContentViewerInteraction` is a bad thing, but it's only available for the huge accessibility Dynamic Type-values.

The font itself doesn't change: 

```swift
button.titleLabel?.font = .monospacedDigitSystemFont(ofSize: 12, weight: .regular)
```

File: MastodonSDK/Package.resolved (hunk 1 above)
Comment (j-f1) [reply MastodonSDK/Package.resolved]: Yes, I believe so — Xcode did this automatically.

File: MastodonSDK/Sources/MastodonUI/View/Control/ActionToolbarContainer.swift (hunk 3 above)
Comment (j-f1) [reply MastodonSDK/Sources/MastodonUI/View/Control/ActionToolbarContainer.swift]: Correct, as I said in the OP the images cannot be easily resized in UIKit, but I could just resize the text.

File: MastodonSDK/Sources/MastodonUI/View/Control/ActionToolbarContainer.swift (hunk 2 above)
Comment (j-f1) [reply MastodonSDK/Sources/MastodonUI/View/Control/ActionToolbarContainer.swift:161]: The button no longer exists so I removed the dead code.

File: MastodonSDK/Sources/MastodonUI/View/Control/ActionToolbarContainer.swift (hunk 3 above)
Comment (j-f1) [reply MastodonSDK/Sources/MastodonUI/View/Control/ActionToolbarContainer.swift]: Update: figured out how to get the images to resize, but having some fun getting the icons to break onto 2 lines for larger text sizes while still retaining the current look on smaller text sizes

**Comment by j-f1** [comment]
Got it sorted partially thanks to https://mastodon.social/@dadederk@iosdev.space/110044528429919745!

**Review by kimar — APPROVED** [review]
LGTM 💯

**Pull request merged**